src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -674,6 +674,9 @@ static QString fontStyleToXML(const FontStyle style, bool allowUnderline = true)
     if (allowUnderline && style & FontStyle::Underline) {
         res += " underline=\"1\"";
     }
+    if (allowUnderline && style & FontStyle::Strike) { // TODO: separate option for allowStrike?
```

Comment:
Not sure we need such an `allowStrike`? @lvinken ?

Reply:
Yes we do. The MusicXML font-style data type used in e.g. fingering and pluck allows only the values normal and italic. We have to be careful not to generate invalid MusicXML.

Reply:
If so the current `allowUnderline` check is sufficient, as it excludes both, `underline` and `line-through`, if `allowUnderline` is set to `false`. We'd need an `allowStrike` only if a `line-through` is forbidden in cases where an `underline` is not. As far as I understand you, such a case doesn't exist, right?

Reply:
Correct

Reply:
So your 1st answer should have been "No, not needed" ;-)